app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -77,15 +77,22 @@ def ping
       render json: { status: 'OK' }
     end
 
-    api :PUT, "/organizations/:organization_id/simple_content_access/enable",
+    api :GET, "/organizations/:organization_id/upstream_subscriptions/simple_content_access/eligible",
```

Comment:
I debated having a separate endpoing (as it is here - `/eligible`) vs just a GET for `/simple_content_access` which we could later, for whatever reason, add more data in the response as we need. That may never happen, so I opted for this and I'm interested in feedback here

Reply:
I like the way you're thinking, but I disagree in this case. As devs you and I know that this set of APIs does talk to hosted candlepin. However, they would not change their interface at all if hosted Candlepin were rewritten in JS (please no) or if the Candlepin API scheme totally changed. Terms like 'Upstream Subscriptions' and 'Simple Content Access' are broader. When we made this controller we needed to differentiate between subscriptions activities happening in the Katello org and the Upstream org, so that's where the name came from. If I added this API under `/katello/api/v2/organizations/simple_content_access/eligible` the question that the API is asking has changed away from "is this **upstream** organization eligible for SCA"

Also keep in mind that my tweak on this line is not changing the API endpoint - it's correcting the apipie doc to match the real API.